fdbserver/RestoreLoader.actor.cpp

```diff
@@ -1007,99 +983,3 @@ ACTOR Future<Void> handleFinishVersionBatchRequest(RestoreVersionBatchRequest re
 	req.reply.send(RestoreCommonReply(self->id(), false));
 	return Void();
 }
-
-// Test splitMutation
-TEST_CASE("/FastRestore/RestoreLoader/splitMutation") {
```

Comment:
Thank you again for having this done! I really appreciate it!
I'm sorry I'm going to ask a lot below:

I feel that there is still value to cross-validate the splitMutation() function. 
Any subtle bug in the function will become very painful to debug. 

Can we use the old implementation to cross-validate the new splitMutation() function? After the fast restore is in prod., we can remove the testing code.

Reply:
Great! Thank you very much!